src/data/actions/userSubscription.js

```diff
@@ -26,13 +26,14 @@ const sendUserSubscriptionFailure = error => ({
 });
 
 const addLicensesForUsers = ({
-  payload,
+  options,
+  subscriptionUUID,
   onSuccess = () => {},
   onError = () => {},
 }) => (
   (dispatch) => {
     dispatch(sendSubscribeUsersRequest());
-    return addLicensesForUsersEndpoint(payload).then((response) => {
+    return LicenseManagerApiService.licenseAssign(options, subscriptionUUID).then((response) => {
```

Comment:
Can we add the UUID in options and take it out of options here? e.g. delete options['subscriptionUUID']

Reply:
`options` includes all the parameters that we are passing to the backend. That's the reason we are passing subscription UUID separately to the service.